mlir/test/mlir-miopen-driver/no_ultra_large_kernels.mlir

```diff
@@ -0,0 +1,3 @@
+// XFAIL: *
```

Comment:
Is there a centralized location for applicability related tests?

Reply:
I'm not aware of one.

Reply:
If that's the case could you add one? Either a folder or a `applicability.mlir` would be fine.

Reply:
Added in https://github.com/ROCmSoftwarePlatform/llvm-project-mlir/pull/381/commits/dd1fcb6ce83199a7a35c0b548f3333204d3b86a7